lib/EpicPlayer.js

```diff
@@ -10,14 +10,30 @@ Structure.extend(
         this.twentyFourSeven = false;
       }
 
+      /**
+       * Sets the resume message as a property of the player so it can be fetched on pause
+       * @param {Message} message
+       */
+      setResumeMessage(message) {
+        if (this.pausedMessage) this.pausedMessage.delete();
+        return (this.resumeMessage = message);
+      }
+
+      /**
+       * Sets the pasued message as a property of the player so it can be fetched on resume
+       * @param {Message} message
+       */
+      setPausedMessage(message) {
+        if (this.resumeMessage) this.resumeMessage.delete();
```

Comment:
Deletion of resume message should happen based on timeout not based on setPausedMessage.

Reply:
I don't see a problem about this method having to delete the old message before setting the new one. Is there any misbehavior which is why we need some delay before deleting the old message?